boards/st/stm32h750b_dk/stm32h750b_dk.dts

```diff
@@ -5,288 +5,29 @@
-		     &ltdc_r4_pj3 &ltdc_r5_pj4 &ltdc_r6_pj5 &ltdc_r7_pj6
-		     &ltdc_g0_pj7 &ltdc_g1_pj8 &ltdc_g2_pj9 &ltdc_g3_pj10
-		     &ltdc_g4_pj11 &ltdc_g5_pi0 &ltdc_g6_pi1 &ltdc_g7_pk2
-		     &ltdc_b0_pj12 &ltdc_b1_pj13 &ltdc_b2_pj14 &ltdc_b3_pj15
-		     &ltdc_b4_pk3 &ltdc_b5_pk4 &ltdc_b6_pk5 &ltdc_b7_pk6
-		     &ltdc_de_pk7 &ltdc_clk_pi14 &ltdc_hsync_pi12 &ltdc_vsync_pi9>;
-	pinctrl-names = "default";
-
-	disp-on-gpios = <&gpiod 7 GPIO_ACTIVE_HIGH>;
-
-	ext-sdram = <&sdram2>;
-	status = "okay";
-
-	clocks = <&rcc STM32_CLOCK(APB3, 3)>,
-		 <&rcc STM32_SRC_PLL3_R NO_SEL>;
-
-	width = <480>;
-	height = <272>;
-	pixel-format = <PANEL_PIXEL_FORMAT_RGB_565>;
-
-	display-timings {
-		compatible = "zephyr,panel-timing";
-		de-active = <1>;
-		pixelclk-active = <0>;
-		hsync-active = <0>;
-		vsync-active = <0>;
-		hsync-len = <1>;
-		vsync-len = <10>;
-		hback-porch = <43>;
-		vback-porch = <12>;
-		hfront-porch = <8>;
-		vfront-porch = <4>;
-	};
-
-	def-back-color-red = <0xFF>;
-	def-back-color-green = <0xFF>;
-	def-back-color-blue = <0xFF>;
-};
-
-&pll {
-	div-m = <5>;
-	mul-n = <192>;
-	div-p = <2>;
-	div-q = <4>;
-	div-r = <4>;
-	clocks = <&clk_hse>;
-	status = "okay";
-};
-
-&pll3 {
-	div-m = <5>;
-	mul-n = <192>;
-	div-p = <2>;
-	div-q = <20>;
-	div-r = <99>;
-	clocks = <&clk_hse>;
-	status = "okay";
-};
-
-&rcc {
-	clocks = <&pll>;
-	clock-frequency = <DT_FREQ_M(480)>;
-	d1cpre = <1>;
-	hpre = <2>;
-	d1ppre = <2>;
-	d2ppre1 = <2>;
-	d2ppre2 = <2>;
-	d3ppre = <2>;
-};
-
-&usart3 {
-	pinctrl-0 = <&usart3_tx_pb10 &usart3_rx_pb11>;
-	pinctrl-names = "default";
-	current-speed = <115200>;
-	status = "okay";
-};
-
-&quadspi {
-	pinctrl-names = "default";
-	pinctrl-0 = <&quadspi_clk_pf10 &quadspi_bk1_ncs_pg6
-		     &quadspi_bk1_io0_pd11 &quadspi_bk1_io1_pf9
-		     &quadspi_bk1_io2_pf7 &quadspi_bk1_io3_pf6
-		     &quadspi_bk2_io0_ph2 &quadspi_bk2_io1_ph3
-		     &quadspi_bk2_io2_pg9 &quadspi_bk2_io3_pg14>;
-	dual-flash;
-	status = "okay";
-
-	/* Sector erase 64KB uniform granularity */
-	/* Subsector erase 4KB, 32KB granularity */
-	mt25ql512ab1: qspi-nor-flash-1@0 {
-		compatible = "st,stm32-qspi-nor";
-		reg = <0>;
-		size = <DT_SIZE_M(512)>; /* 512 Mbits */
-		qspi-max-frequency = <72000000>;
-		cs-high-time = <4>; /* >= 50 ns */
-		spi-bus-width = <4>;
-		reset-cmd;
-		status = "okay";
-
-		partitions {
-			compatible = "fixed-partitions";
-			#address-cells = <1>;
-			#size-cells = <1>;
-
-			slot0_partition: partition@0 {
-				label = "image-0";
-				reg = <0x00000000 DT_SIZE_K(2048)>;
-			};
-
-			slot1_partition: partition@200000 {
-				label = "image-1";
-				reg = <0x00200000 DT_SIZE_K(2048)>;
-			};
-
-			storage_partition: partition@400000 {
-				label = "storage";
-				reg = <0x00400000 DT_SIZE_K(128)>;
-			};
-		};
-	};
-
-	mt25ql512ab2: qspi-nor-flash-2@0 {
-		compatible = "st,stm32-qspi-nor";
-		reg = <0>;
-		size = <DT_SIZE_M(512)>; /* 512 Mbits */
-		qspi-max-frequency = <72000000>;
-		status = "okay";
-	};
```

Comment:
Suppression of this bank should be at least documented and explain in the commit message.